rust-toolchain.toml

```diff
@@ -1,5 +1,6 @@
 # If you modify this file, please also update `RUST_TOOLCHAIN` in `.github/workflows/ci.yml`, the
-# `clippy_utils` version in `bevy_lint/Cargo.toml`, and the toolchain in `README.md`.
+# `clippy_utils` version in `bevy_lint/Cargo.toml`, the toolchain in `README.md`, and
+# `RUST_TOOLCHAIN` in `bevy_lint/src/bin/main.rs`.
```

Comment:
Feels like this is going to become a bit of a chore. Should we be using `rust-toolchain.toml` as the single source of truth?

Reply:
Yup! I addressed this, so now all that's left is the `clippy_utils` version (which can't be removed, since it requires manually finding an appropriate Git hash).

I did have to use build scripts to embed the toolchain version in `bevy_lint/src/bin/main.rs`, though I don't think it's too bad.